Assets/Scripts/UI/PickupPup/DogSlot/DogSlot.cs

```diff
@@ -91,8 +91,11 @@ protected override void setReferences()
 	{
 		base.setReferences();
 		button = ensureReference<UIButton>(searchChildren:true);
-		subscribeToUIButton();
-	}
+        images = GetComponentsInChildren<Image>();
+        dogImage = images[1];
```

Comment:
Funny thing is, this was already in the code. I just moved it to a different line. Will fix anyway.